src/services/auth.js

```diff
@@ -0,0 +1,41 @@
+import request from '../utils/request';
+
+const { endpoints } = window;
+
+export const queryRegisterUser = async params => {
+  const endpoint = `${endpoints.pbench_server}/user`;
```

Comment:
Isn't this `${endpoints.pbench_server}/register` in the code we merged? Are you working with an old server install?

Reply:
Currently working on updating it